jest/setup/react-dom.js

```diff
@@ -0,0 +1,9 @@
+import ReactDOM from 'react-dom'
+
+beforeAll(() => {
+  ReactDOM.createPortal = jest.fn(el => el)
```

Comment:
why do this?

Reply:
so i didn't have to skip the storyshots. jsdom(or whatever jest is using) doesn't work with portals but this way i found we can just render the portal's content back and still get the snapshot